### ISSUE TITLE
3.23.96 (Desktop) Can't resize middle column

### ISSUE THREAD

**Issue by catwithbanana** [standardnotes/app#1425]
After the latest update the middle column (with the list of notes) can no longer be resized. On its right side there are two different "grab points" now. One shrinks the text editing column without expanding the notes list column, leaving an empty space. The other moves the entire window around.

The notes list column is stuck being extremely narrow, causing all sorts of strange text overflow. Here's a small screenshot to show the problem area:
Narrow column:
<img width="480" alt="image" src="https://user-images.githubusercontent.com/75995110/186253942-f818c534-44c3-4fbd-8bf0-b0f32907192d.png">

Shrinking text edit field doesn't expand Notes list to fill space:
<img width="548" alt="image" src="https://user-images.githubusercontent.com/75995110/186254019-45412ea5-12be-486d-964f-a10081eea8b6.png">

**Comment by kydn** [comment]
I can reproduce. standard-notes-3.23.96-mac-arm64

**Comment by ghughes** [comment]
Seeing the same issue here after updating to 3.23.96 on macOS 12.3.1. Deleting `~/Library/Application Support/Standard Notes` doesn't help. [Here's a screen recording that shows the two separate "grab points".](https://www.dropbox.com/s/q3yl42653x47k8e/Standard%20Notes%20width%20bug.mov?dl=0)

**Comment by ghughes** [comment]
(The following was already discussed in Discord; repeating here for the record)

While dragging, I see the inline style for #items-column being updated with the new width, but this rule is overriding it:
```css
@media (min-width: 768px) and (max-width: 1023px) {
  .md-only\:\!w-52 { width: 13rem !important; }
}
```

The problem is resolved if you set `width: 100% !important` on the column and *then* resize it, which makes me suspicious of [this](https://github.com/standardnotes/app/commit/f555fbcaaab1203c8460e7a69c3d7723d09f45d7#diff-bb8d74503edbc7f6591f3d4adaf751a7146801a33e17d8bb913c572ceb00763dL20) recent change.

A temporary workaround is to adjust the SN window to >1279px wide.

**Comment by effieeee** [comment]
Fix for this is coming up soon 😊 #1497
